.gitignore

```diff
@@ -1 +1,2 @@
 /config/database.yml
+/log/development.log
```

Comment:
please add this ignores too:
# Ignore bundler config.
/.bundle

# Ignore all logfiles and tempfiles.
/log/*
!/log/.keep
/tmp
/.idea
/.rvmrc